src/store/proposals/proposals.slice.ts

```diff
@@ -28,6 +30,19 @@ export const appSlice = createSlice({
       state.proposalsList.data = payload;
       state.proposalsList.state = 'success';
     },
+    startProposal: (state, _: PayloadAction<AddProposalFields>) => {
```

Comment:
No need to use the second argument here 
```suggestion
    startProposal: (state) => {
```

Reply:
This argument is not needed in the slice, but it's there in typescript purpose, to set the proper type of action payload which is later used in the saga

Reply:
My bad then, missed that